web-server/src/components/TeamSelector/TeamPopover.tsx

```diff
@@ -323,6 +330,15 @@ export const TeamPopover: FC<{
                   'We getting your teams together, but someone seems missing 🤔'
                 )}
               </Scrollbar>
+              {Boolean(apiTeams.length) && (
+                <FlexBox marginTop={'10px'}>
```

Comment:
make this `mt={2}`

<img width="251" alt="image" src="https://github.com/user-attachments/assets/ba258d57-1b3e-4cfa-9a20-a10ee8c01be4">

looked better

Reply:
Sure